code/Util/StringExtensions.cs

```diff
@@ -4,6 +4,8 @@ namespace TTT;
 
 public static class StringExtensions
 {
+	public static bool IsNullOrEmpty( this string s ) => s is null || s.Length == 0;
```

Comment:
Honestly, I don't like having this extension. I think we should keep using the default `string.IsNullOrEmpty()` check, even though it's flawed.

Reply:
it matches the same `List` extension we use, seems a bit odd to break standards.

Reply:
> it matches the same `List` extension we use, seems a bit odd to break standards.

you're breaking c# standards by not using what everyone else is using 🥴

Reply:
Please point me towards `List.IsNullOrEmpty()`

Reply:
> Please point me towards `List.IsNullOrEmpty()`

where

Reply:
🥴

Reply:
We can change it but just realize that we have other instances where we do null checks that follow the exact same pattern.

ex. `IsValid()` <- s&box api
ex. `List.IsNullOrEmpty()`
ex. `Panel.IsEnabled()`

It makes sense to just write this single util and start using it.